src/NuGet.Clients/NuGet.SolutionRestoreManager/VSNominationUtilities.cs

```diff
@@ -151,6 +152,17 @@ internal static RuntimeGraph GetRuntimeGraph(IEnumerable targetFrameworks)
             return tfi;
         }
 
+        internal static NuGetFramework GetTargetFramework(IVsProjectProperties properties)
+        {
+            var targetFrameworkIdentifier = GetPropertyValueOrNull(properties, ProjectBuildProperties.TargetFrameworkIdentifier) ?? string.Empty;
```

Comment:
I think you should use the same utility I wrote in https://github.com/NuGet/NuGet.Client/pull/3562. 

This way, UAP projects will fail. 

Now I know UAP projects are not supported with SDK based projects, but customers can still do it, so we sohuld consider it.

Reply:
First person who merges their PR wins, and the other person has to update their PR to include this change 😄

Reply:
I merged the PR....I can rebase this PR for you if you want me...or you can quickly do it yourself :)